### ISSUE TITLE
I have an error in android v.9 (pie) when I get current location

### ISSUE THREAD

**Issue by dyazincahya** [NativeScript/nativescript-geolocation#236]
### my code
  ``` javascript
var location = geolocation.getCurrentLocation({desiredAccuracy: 3, updateDistance: 10, maximumAge: 20000, timeout: 20000}).
  then(function(loc) {
      if (loc) {
        getKoordinat = true;
        myloc=loc;
        model.getParentMap(page.navigationContext.data['rp_id'],myloc).then(function(result2){
          console.log("masuk log marker");
          loadmarker(result2);
        });
        if(mapView){
          mapView.latitude = myloc.latitude;
          mapView.longitude = myloc.longitude;  
          mapView.zoom = 17;
        }
      }
  }, function(e){
      console.log("Error: " + e.message);
  });
```

### Which platform(s) does your issue occur on?
Android versions 

### Please, provide the following version numbers that your issue occurs with:
NS v.3

### Error
An uncaught Exception occurred on "Thread-8" thread.
java.lang.NoClassDefFoundError: Failed resolution of: Lorg/apache/http/ProtocolVersion;
	at gv.b(:com.google.android.gms.dynamite_mapsdynamite@18382081@18.3.82 (100400-260264002):3)
	at gw.a(:com.google.android.gms.dynamite_mapsdynamite@18382081@18.3.82 (100400-260264002):3)
	at gy.a(:com.google.android.gms.dynamite_mapsdynamite@18382081@18.3.82 (100400-260264002):14)
	at com.google.maps.api.android.lib6.drd.aj.a(:com.google.android.gms.dynamite_mapsdynamite@18382081@18.3.82 (100400-260264002):6)
	at ga.a(:com.google.android.gms.dynamite_mapsdynamite@18382081@18.3.82 (100400-260264002):22)
	at ga.run(:com.google.android.gms.dynamite_mapsdynamite@18382081@18.3.82 (100400-260264002):8)
Caused by: java.lang.ClassNotFoundException: Didn't find class "org.apache.http.ProtocolVersion" on path: DexPathList[[zip file "/data/user_de/0/com.google.android.gms/app_chimera/m/0000002c/MapsDynamite.apk"],nativeLibraryDirectories=[/data/user_de/0/com.google.android.gms/app_chimera/m/0000002c/MapsDynamite.apk!/lib/armeabi-v7a, /data/user_de/0/com.google.android.gms/app_chimera/m/0000002c/MapsDynamite.apk!/lib/armeabi, /system/lib]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at ad.loadClass(:com.google.android.gms.dynamite_dynamiteloader@18382081@18.3.82 (100400-260264002):4)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	... 6 more

**Comment by hitmacreed** [comment]
Hi, if u are using android 9 at this to your manifest

**Comment by elena-p** [comment]
Hi @dyazincahya,

It seems that you are using very old version of NativeScript, which is no longer supported. We have recently released {N} 6.1. I would highly encourage you to update your app to use latest version of the framework, as this would highly improve your chances of getting help from the community or the NativeScript team.

**Comment by dyazincahya** [comment]
```
D:\project\nativescript\fiberstar\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:21: AAPT: error: unknown element <uses-library> found.


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Failed to process resources, see aapt output above for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 37s
Unable to apply changes on device: 3045a4480704. Error is: Command gradlew.bat failed with exit code 1.
```

I have an error when I add this tag 
```
<uses-library android:name="org.apache.http.legacy" android:required="false"/>
```
